### ISSUE TITLE
[Idea] Ability to tag a profile 

### ISSUE THREAD

**Issue by Jambrose777** [thewca/worldcubeassociation.org#5983]
**Is your feature request related to a problem? Please describe.**
Sometimes the Results team, WRC, and Delegates have to await a response from a competitor for some issue. This can be something wrong with their personal information, an issue with an incident or result, etc. Often times if the competitor never responds we make the best logical conclusion that we can, though it may not be necessarily correct. 

**Describe the solution you'd like**
It would be nice to be able to tag these competitors for if/when they register for another competition the Delegate / relevant party knows about it and can further get answers via email or in person at that competition.

Just an idea, would like to hear others thoughts on such a feature.

**Comment by AlexKatyukov** [comment]
**TL;DR: yes, please!**

It is good idea to have an opportunity to tag the profile with comments in some cases, for example if there are some suspicions about incorrect personal data or if seems that competitor cheat but it is not proven to warn Delegates of next competitions about it. Maybe there are other issues where that feature can help.

Will be good if there will be some privacy settings about who can see tags. For example, can info be seen by organizers of future competitions? If it is info about incorrect data it is okay; if it is info about unproven cheating — no. Maybe teams/committees needs some internal labels.

**Comment by Jambrose777** [comment]
> Will be good if there will be some privacy settings about who can see tags. For example, can info be seen by organizers of future competitions? If it is info about incorrect data it is okay; if it is info about unproven cheating — no. Maybe teams/committees needs some internal labels.

For everything I can imagine we'd use this feature on, we wouldn't want organizers involved. Everything with personal data, results, and especially internal WCA communication, the WCA Delegate is ultimately responsible. An organizer wouldn't have the skillset to handle these types of issues. So I would not recommend organizers having access to see who's tagged and for what.

**Comment by SAuroux** [comment]
Interesting idea! 

What comes to my mind spontaneously for a quite simple realization is that tagging a profile could be an admin-level feature, so that adding/removing tags is only possible for admins and WRT, maybe WDC and WEC too. Such a person registering for a competition could then trigger an email to be sent to WRT and competition delegates to immidiately discuss corresponding matter matter.

**Comment by saranshgrover** [comment]
Marking low-priority for now until someone would like to work on it/explore further
